lib/src/git_backend.rs

```diff
@@ -341,9 +343,13 @@ impl GitBackend {
 
 fn commit_from_git_without_root_parent(
     id: &CommitId,
-    commit: &gix::objs::CommitRef,
+    git_object: &gix::Object<'_>,
```

Comment:
nit: can the lifetime be elided?

Reply:
Sure, strange that clippy didn't yell at me for that